draft-ietf-rats-msg-wrap.md

```diff
@@ -641,6 +641,14 @@ The initial registrations for the registry are detailed in {{tab-ind-regs}}.
 | 3 | Attestation Results | {{&SELF}} |
 {: #tab-ind-regs title="CMW Indicators Registry Initial Contents"}
 
+### Provisional Registration
+
+Before the creation of the registry by IANA, new codepoints can be added to the [provisional CMW Indicators registry](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/blob/provisional-registry/provisional/cmw-indicators-registry.md) by following the documented procedure.
```

Comment:
**NOTE** Before merging the URL needs to be changed to match the final location (i.e., `main` rather than the `provisional-registry` branch)